backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -173,6 +172,10 @@ void postConstruct() {
                     // Delete on daily
                     aggregationCommand.getSubscriptionType().equals(EmailSubscriptionType.DAILY)
             ).onItem().ignoreAsUni();
+        })
+        .onFailure().recoverWithUni(t -> {
+            LOGGER.info("Error while processing aggregation", t);
```

Comment:
Same logging level as what we have in [EventConsumer](https://github.com/RedHatInsights/notifications-backend/blob/614f71f6e7ab660bf4ffea07c650c9266f2c0300/backend/src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java#L178).